lib/measured/util.rb

```diff
@@ -0,0 +1,24 @@
+module Measured
+  class << self
+    alias_method :old_build, :build
```

Comment:
Rather than method alias chaining, try module prepend:
http://www.justinweiss.com/articles/rails-5-module-number-prepend-and-the-end-of-alias-method-chain/